lib/engine-addon.js

```diff
@@ -8,6 +8,7 @@ var writeFile = require('broccoli-file-creator');
 var Babel = require('broccoli-babel-transpiler');
 var concat = require('broccoli-concat');
 var DependencyFunnel = require('broccoli-dependency-funnel');
+var defaults = require('lodash.defaults');
```

Comment:
Agreed all around. I had forgotten that we required Ember-cli@2.12 with ember-engines@0.5.0. Seems like 0.5 dropped node 0.12 support for us. 🎊